streamline flytesnacks tutorials (#135)
flyteorg/flytesnacks: cookbook/docs/index.rst

Flyte Tutorials
==============================================

> These tutorials are intended to be a learn by example style cookbook.  We cover all the various concepts in Flyte along with most of the maintained

Reviewer: ```suggestion
These tutorials are intended to help the user learn by example.  We cover all the various concepts in Flyte along with most of the maintained
```

> plugins. This cookbook is designed to get you running both locally, and on a Flyte cluster using `Flytekit Python <https://github.com/lyft/flytekit>`__

Reviewer: ```suggestion
plugins. All the examples are authored using `Flytekit Python <https://github.com/lyft/flytekit>`__ and are designed to be run-locally (this may not be possible for some plugins) or on a Flyte cluster 
```
